packages/commonwealth/server/migrations/20240621195630-rotate-alchemy-keys.js

```diff
@@ -10,8 +10,9 @@ module.exports = {
       process.env.NEW_ALCHEMY_PUBLIC_KEY || process.env.ETH_ALCHEMY_API_KEY;
 
     if (!privateKey || !publicKey) {
-      throw new Error(
```

Comment:
I am not sure how we should handle these missing keys. I changed it to be a warning becuase we will not be giving external developers these keys, but maybe someone has a better idea?

Reply:
Maybe I can update the readme with things that will be broken? @timolegros could you chime in here, as I am not sure the extent of the features that uses the chainNode

Reply:
Chain requests will fail for the major EVM chains so devs won't be able to use groups, stake, contests, etc locally. The instructions should make it clear that devs should get their own Alchemy keys if they want to interact with chain features on Common EVM communities.

I don't think we should provide a placeholder because as you said this would propagate the error downstream.

Reply:
Well we should decide whether to have placeholder or not. Basically the end result will be the same, that is the error will not be thrown on these migrations that require this API key. The pros/cons of using the placeholder:

PROs:
* Don't need to change old migrations

CONs:
* Don't get a warning message when these migrations are ran

But I think the CON is fine if we clearly lay out in the README that this API key is needed for the chain functionality to work.

Reply:
As discussed during parking lot, we will go with changing the migration